### DISCUSSION TITLE
Toggle On/Off Text label or Size or All Caps

### DISCUSSION THREAD

**Discussion by simonaglover95** [Pirate-MIDI/Pirate-MIDI-Features-Bug-Tracking#51] [Feature Requests & Ideas]
It would be nice if we could put different text for switch names when using toggle on/off. This would also aid in knowing if it is toggled on or off.

Having two text fields, one for toggle on and another for off would solve this.

Another option could be a font size change. Bigger letters when toggled on. This would consume more screen space but there is plenty to steal from over/under "Bank name".

**Comment by simonaglover95** [comment]
Originally submitted on looped.in ideas page with 15 community votes

**Comment by ultracello** [comment]
Good ideas. Both ones.
